strax/mailbox.py

```diff
@@ -203,7 +203,7 @@ def cleanup(self):
         for t in self._threads:
             t.join(timeout=self.timeout)
             if t.is_alive():
-                raise RuntimeError("Thread %s did not terminate!" % t.name)
+                raise RuntimeError("Thread %s did not terminate! %s" % (t.name,self.timeout))
```

Comment:
Would you like to make me happy and change this also into a f-string? Also could you please add some additional info about self.timeout into the string?